src/plugins/data/public/antlr/promql/promql_autocomplete.ts

```
@@ -0,0 +1,171 @@
+): ProcessPromQLVisitedRulesResult<PromQLAutocompleteResult> {
+  let suggestMetrics = false;
+  let shouldSuggestLabels;
+  let shouldSuggestLabelValues = false;
+  let suggestTimeRangeUnits = false;
+  let suggestAggregationOperators = false;
+  let suggestFunctionNames = false;
+
+  for (const [ruleId, rule] of rules) {
+    switch (ruleId) {
+      case PromQLParser.RULE_aggregationOperators:
+        suggestAggregationOperators = true;
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)